src/components/beers/List.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+
+export const List = ({ children }: { children: React.ReactNode }) =>
+  <ul
+    className={
+      'mx-auto grid max-w-screen-xl grid-cols-1 justify-items-stretch gap-2 sm:grid-cols-2 lg:grid-cols-3'
+    }
+  >
+    {children}
+  </ul>
+
+interface ListItemModel {
+    children: React.ReactNode
+    classList?: string
+}
+
+export const ListItem = ({
```

Comment:
jakbym to pisał sam to dałbym to do oddzielnego pliku, ale plik i tak jest krótki i to raczej subiektywne, także powiedzmy ze to luźna sugestia 😃